src/devices/ExplorerHat/Led.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat
+{
+    /// <summary>
+    /// Represents a led light
+    /// </summary>
+    public class Led : IDisposable
+    {
+        private GpioController _controller;
+
+        /// <summary>
+        /// GPIO pin to which led is attached
+        /// </summary>
+        public int Pin { get; private set; }
+
+        /// <summary>
+        /// Led number on Hat
+        /// </summary>
+        public int Number { get; private set; }
```

Comment:
Is Number and Name needed for anything specific?

Reply:
Not now, that's right. But I'm thinking about make class "Ligths" IEnumerable, so Name (would be renamed to Colour) and Number will help you to identify them when you randomly pick them up. What dou you think?

Reply:
@Kash0321 I'm a bit skeptical about adding stuff which is not immediately obvious what's for. I.e. imagine adding a "Name" property to Stream so that you know which stream is what.. Normally you'd create some kind of dictionary with (string => Stream) - while you can use properties I'd argue that should belong to the view model of your light and not to the hardware implementation.